_includes/head-header.html

```diff
@@ -494,6 +509,10 @@
     </script>
 </head>
 <body>
+<!-- Google Tag Manager (noscript) -->
+<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TGBNQXN"
+                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+<!-- End Google Tag Manager (noscript) -->
```

Comment:
https://support.google.com/tagmanager/answer/6103696?hl=en#install&zippy=%2Cwhat-is-an-account